app/_assets/styles/custom/pages/_page-home.scss

```diff
@@ -392,17 +392,20 @@ $home-feature-icon-bottom-spacing: 1rem;
 
       code[class*='language-'],
       pre[class*='language-'] {
-        color: #fff;
+        color: #ffffffdd;
```

Comment:
Many thanks for this improvement @fuchen-shi 🙂 

Just a nit on the `color` here:
Instead of modifying the alpha channel (transparency/opacity) to achieve a slightly darker result, I'd suggest to make use of our predefined [gray-scales as SCSS variables](https://github.com/kumahq/kuma-website/blob/5b5b301e0c92067cff56a179639139fa8d8632f9/app/_assets/styles/custom/config/_variables.scss#L29). How about using `$gray-4` for this?

Reply:
Of course! I've updated the color to use the variable.

Reply:
Thanks! 👍